src/test/scala/nl/knaw/dans/easy/schemaExamples/DdmSchemaSpec.scala

```diff
@@ -15,18 +15,42 @@
  */
 package nl.knaw.dans.easy.schemaExamples
 
+import better.files.File
 import org.scalatest.prop.TableFor1
 
+import scala.util.Failure
+
 class DdmSchemaSpec extends SchemaValidationFixture {
 
   override val publicSchema: String = s"$httpsEasySchemaBase/md/ddm/ddm.xsd"
   override val localSchemaFile: String = lastLocalXsd("md", "ddm.xsd")
-  override val examples: TableFor1[String] = Table(
-    "example",
-    "dcx-dai/example1.xml",
-    "dcx-gml/example1.xml",
-    "ddm/example1.xml",
-    "ddm/example2.xml",
-    "abr-type/example1.xml",
+  override val examples: TableFor1[File] = Table("file",
+    exampleDir / "dcx-dai/example1.xml",
+    exampleDir / "dcx-gml/example1.xml",
+    exampleDir / "ddm/example1.xml",
+    exampleDir / "ddm/example2.xml",
+    exampleDir / "abr-type/example1.xml",
   )
+
+  "relation validation" should "report an invalid DOI" in pendingUntilFixed {
+    validateWithLocal(modify(
+      ".*</ddm:relation>.*",
+      """<ddm:relation scheme="id-type:DOI" href="https://doi.org/42">42</ddm:relation>"""
+    )) shouldBe a[Failure[_]]
+  }
+
+  it should "report an XSS attack" in pendingUntilFixed {
+    validateWithLocal(modify(
+      ".*</ddm:relation>.*",
+      """<ddm:relation xml:lang="nld" href="javascript:alert('XSS')">xxx</ddm:relation>"""
```

Comment:
Pending as this test is actually implemented as a rule by https://github.com/DANS-KNAW/easy-validate-dans-bag/pull/74
but it could possibly also (partly) done with a regexp.